sbt-web-scalajs-bundler/src/sbt-test/sbt-web-scalajs-bundler/play/server/src/main/scala/example/Loader.scala

```diff
@@ -10,7 +10,10 @@ class Loader extends ApplicationLoader {
 }
 
 class ExampleComponents(context: Context) extends BuiltInComponentsFromContext(context) {
+  val httpFilters = Seq()
   val controller = new ExampleController
-  val assets = new Assets(httpErrorHandler)
+  val assetsConfiguration = new AssetsConfiguration()
+  val assetsMetadata = new DefaultAssetsMetadata(assetsConfiguration, environment.resource _, fileMimeTypes)
```

Comment:
There should be `XxxComponents` traits to mix in to our class so that we don’t have to manually instantiate the `AssetsMetadata`, `AssetsConfiguration` and the `HttpFilter` list.